src/widget/wstarrating.h

```diff
@@ -22,16 +22,13 @@ class WStarRating : public WWidget {
     virtual void setup(const QDomNode& node, const SkinContext& context);
     QSize sizeHint() const override;
 
-    /// Manually set a custom rating
-    ///
-    /// The value must be consistent with the current track if connected.
-    void setRating(int rating);
-
   public slots:
-    void slotTrackLoaded(TrackPointer pTrack = TrackPointer());
+    void slotSetRating(int rating);
+
+  signals:
+    void ratingChangeRequest(int rating);
```

Comment:
Events are named using past tense, i.e. `ratingChanged()`. For requests (method or command) you use an imperative verb.

Reply:
:+1: